lib/pengine/bundle.c

```diff
@@ -1005,6 +1005,10 @@ pe__unpack_bundle(pcmk_resource_t *rsc, pcmk_scheduler_t *scheduler)
     } else {
         xml_obj = first_named_child(rsc->xml, PCMK_XE_RKT);
         if (xml_obj != NULL) {
+            pcmk__warn_once(pcmk__wo_rkt,
```

Comment:
Consider also making `PCMK_XE_RKT` an internal constant and adding a COMPAT note. This constant was just added this month, so it's not in any release yet.

Reply:
good point